plugins/kotlin/j2k/shared/src/org/jetbrains/kotlin/nj2k/conversions/StreamConversion.kt

```diff
@@ -0,0 +1,38 @@
+// Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package org.jetbrains.kotlin.nj2k.conversions
+
+import org.jetbrains.kotlin.analysis.api.KtAnalysisSession
+import org.jetbrains.kotlin.nj2k.NewJ2kConverterContext
+import org.jetbrains.kotlin.nj2k.RecursiveConversion
+import org.jetbrains.kotlin.nj2k.callOn
+import org.jetbrains.kotlin.nj2k.tree.*
+
+class StreamConversion(context: NewJ2kConverterContext) : RecursiveConversion(context) {
+
+    context(KtAnalysisSession)
+    override fun applyToElement(element: JKTreeElement): JKTreeElement {
+        if (element !is JKQualifiedExpression) return recurse(element)
+        if (element.selector !is JKCallExpressionImpl) return recurse(element)
+
+        val target = (element.selector as JKCallExpressionImpl).identifier.fqName
+
+        // TODO: Handle other cases too.
```

Comment:
Do you have some cases in mind?

Reply:
Not exact cases, but I've seen that you have this ticket, which I assume, means the handling of other cases too: https://youtrack.jetbrains.com/issue/KTIJ-14393/Convert-Java-8-stream-API-operations-to-Kotlin-sequence-operations

Reply:
Well, this issue is a small bug that leads to red code. [KTIJ-14393](https://youtrack.jetbrains.com/issue/KTIJ-14393) is a big feature request to map Stream API to Sequence API. So, it will be handled separately.